localboot/grub.go

```diff
@@ -65,6 +76,9 @@ func ParseGrubCfg(grubcfg string, basedir string, grubVersion int) []bootconfig.
 			}
 			inMenuEntry = true
 			cfg = new(bootconfig.BootConfig)
+			name := strings.Join(sline[1:], " ")
+			name = strings.Split(name, "--")[0]
```

Comment:
Is this to skip the optional arguments, I guess? https://www.gnu.org/software/grub/manual/grub/grub.html#menuentry 

This makes a lot of assumptions without taking into account, e.g., quoting. Parsing GRUB configs is a sh**show in general :(
For this specific part, we should improve `unquote` to also check single and double quotes (see https://www.gnu.org/software/grub/manual/grub/grub.html#Quoting  ) and implement a split method that handles quotes.

Reply:
Yor are right. In that way it works at least if singel quotes are present in the line.
Since the name field of the BootConfig struct wasn't set to a meaningfull value at all, I just want to assign something to be able to identify the choosen BootConfig in the log.
As mentioned above a well doing grub parser is an other topic. 
So I think we can go with it for now.